### PR TITLE
Qutip4.4

### PR DESCRIPTION
**Pull request by Ericgig**
Qutip 4.4
Documentation as incomplete change-log

### CODE REVIEW

File: _includes/sidebar.html [hunk 1]
```diff
@@ -4,15 +4,17 @@
...
-        <a onclick="javascript:_gaq.push(['_trackEvent','download','qutip','qutip-4.3.1.zip']); void(0);" href="https://github.com/qutip/qutip/archive/v4.3.1.zip">zip</a>, 
+    <p>July 18, 2018<br />
+        4.4.0 -
+        <a onclick="javascript:_gaq.push(['_trackEvent','download','qutip','qutip-4.4.0.tar.gz']); void(0);" href="downloads/4.4.0/qutip-4.4.0.tar.gz">tar.gz</a>,
+        <!-- href="https://github.com/qutip/qutip/archive/v4.4.0.tar.gz" -->
+        <a onclick="javascript:_gaq.push(['_trackEvent','download','qutip','qutip-4.4.0.zip']); void(0);" href="downloads/4.4.0/qutip-4.4.0.zip">zip</a>,
+        <!-- href="https://github.com/qutip/qutip/archive/v4.4.0.zip" -->
         <a href="docs/latest/index.html">docs</a>
-        (<a onclick="javascript:_gaq.push(['_trackEvent','download','qutip-doc','qutip-doc-4.3.pdf']); void(0);" href="downloads/4.3.1/qutip-doc-4.3.pdf">pdf</a>)
+        (<a onclick="javascript:_gaq.push(['_trackEvent','download','qutip-doc','qutip.pdf']); void(0);" href="downloads/4.4.0/qutip.pdf">pdf</a>)
```
Comment (ajgpitch) [inline _includes/sidebar.html]: Is there some reason we did not call the file qutip-doc-4.4.pdf?

File: documentation.html [hunk 2]
```diff
@@ -30,7 +30,7 @@ <h3>Version 4.3</h3>
 </div>
 <div class="col-md-3 col-md-offset-1">
 <img src="images/pdf.png" />
-<a onclick="javascript:_gaq.push(['_trackEvent','download','qutip-doc','qutip-doc-4.3.pdf']); void(0);" href="downloads/4.3.1/qutip-doc-4.3.pdf">PDF documentation</a>
+<a onclick="javascript:_gaq.push(['_trackEvent','download','qutip-doc','qutip.pdf']); void(0);" href="downloads/4.4.0/qutip.pdf">PDF documentation</a>
```
Comment (ajgpitch) [inline documentation.html]: Again here

**Comment by ajgpitch** [comment]
You'll see I made a PR on the core, so I guess the bdist will need to remade, sorry

File: _includes/sidebar.html (hunk 1 above)
Comment (Ericgig) [reply _includes/sidebar.html]: Not really. That's the filename being build, why rename it?

**Pull request merged**